renaissance-core/src/main/java/org/renaissance/Benchmark.java

```diff
@@ -0,0 +1,90 @@
+package org.renaissance;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+public interface Benchmark {
+
+  /**
+   * Name of the benchmark. Defaults to kebab-case variant of the benchmark's
+   * class name (without the package prefix). For example, the default name for
+   * a class named {@code LittleBigBench} will be {@code little-big-bench}.
+   */
+  @Documented
+  @Retention(RetentionPolicy.RUNTIME)
+  @interface Name {
+
+    String value();
+
+  }
+
+
+  /**
+   * Name of the group the benchmark belongs to. Defaults to dash-separated
+   * package names under org.renaissance. For example, the group name for a
+   * benchmark class in the {@code org.renaissance.jdk.concurrent} package
+   * will default to {@code jdk-concurrent}.
+   */
+  @Documented
+  @Retention(RetentionPolicy.RUNTIME)
+  @interface Group {
+
+    String value();
```

Comment:
Same here. Seems that the default convention is sufficient, and makes things simpler.

Reply:
I would contest the `simpler`, see above. It's a different concern. Am I the only one seeing ghosts? :-)

Reply:
Ok - in that case, I would have a check that the name respects the rule that the class name corresponds to the name, when kebab-cased.

Reply:
As above, I used explicit annotations and will add a build check.